app/src/main/java/net/bonysoft/magicmirror/sfx/GlowView.java

```diff
@@ -0,0 +1,54 @@
+package net.bonysoft.magicmirror.sfx;
+
+import android.content.Context;
+import android.graphics.drawable.ColorDrawable;
+import android.graphics.drawable.Drawable;
+import android.graphics.drawable.TransitionDrawable;
+import android.support.annotation.ColorRes;
+import android.support.v4.util.SparseArrayCompat;
+import android.util.AttributeSet;
+import android.view.View;
+
+public class GlowView extends View {
+
+    private static final int TRANSITION_DURATION = 700;
+
+    private final SparseArrayCompat<ColorDrawable> colors = new SparseArrayCompat<>();
+
+    public GlowView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public void transistToColor(@ColorRes int colorRes) {
```

Comment:
After a long discussion about the English language, TIL I learnt that the verb of transition is... transition. 

God help us